libs/application-generic/src/usecases/create-execution-details/types/index.ts

```diff
@@ -14,7 +14,6 @@ export enum DetailEnum {
   MESSAGE_CONTENT_SYNTAX_ERROR = 'Message content could not be generated due to syntax error in email editor',
   MESSAGE_CREATED = 'Message created',
   SUCCESSFUL_BRIDGE_RESPONSE_RECEIVED = 'Successful Bridge response received',
-  FAILED_BRIDGE_RETRY = 'Failed to get response from Bridge, will be retried',
```

Comment:
Not used anymore.

Reply:
what will happen if, for example, Novu hits 408 Request Timeout on the bridge app?

Reply:
With this change, they will now only see a `Bridge execution failed` error x3. I will add a screenshot to demonstrate.

The current injection of the `afterResponse` hook doesn't expose an interface for reading that a retry occurred due to the retry condition being met, so some rework is necessary to continue using this error code.

I'm not convinced that much value is being added by this `"will be retried"` sentence as the retries occur immediately and will log immediately, so I decided to remove it.

Reply:
Ok, thanks for calling this out.

The extra execution detail logging is in fact needed for reporting the case of `408 Request Timeout`. I'm looking in to why.

Reply:
Resolved, see PR description.